.circleci/config.yml

```diff
@@ -97,6 +97,77 @@ commands:
           event: pass
           template: basic_success_1
 
+  cf_deploy_review:
+    steps:
+      - run:
+          name: "Setup CF CLI"
+          command: |
+            curl -L -o cf.deb --retry 3 'https://packages.cloudfoundry.org/stable?release=debian64&version=7.2.0&source=github-rel'
+            file cf.deb
```

Comment:
Is this for debug?

Reply:
Just duplicated what was already there, but yes - we were getting odd errors from curl - and this was to keep track - though I can come up with a better solution - I'd like to standardise the 'setup CF' step if at all possible, I'll do that as part of a follow up PR